Makefile

```diff
@@ -48,3 +48,15 @@ run:
 
 check:
 	pytest-3 --color=yes --showlocals -vv $(TEST_TARGET)
+
+check-in-container:
+	$(CONTAINER_ENGINE) run \
+		-ti --rm \
+		-v $(CURDIR)/dist2src:/usr/local/lib/python3.6/site-packages/dist2src:Z \
+		-v $(CURDIR)/packitpatch:/usr/bin/packitpatch:Z \
+		-v $(CURDIR)/macros.packit:/usr/lib/rpm/macros.d/macros.packit:Z \
+		-v $(CURDIR)/tests_localhost:/tests_localhost:Z \
+		--entrypoint= \
+		-u $(shell id -u) \
+		$(OPTS) \
+		$(IMAGE_NAME) pytest --color=yes --showlocals -vv /tests_localhost
```

Comment:
Color codes [can look ugly](https://softwarefactory-project.io/logs/43/43/ea1c54c2a4a5092d57767f05d6fb5d99a8fcfc98/check/dist2src-tests-inside-container/66b01a5/job-output.txt.gz) in logs. How about making them optional like it was done [here](https://github.com/packit/packit-service/commit/582b5b7bec369cd6e61ffb45ac2f3d9aa7faf39d)?